service/service.go

```diff
@@ -129,6 +131,20 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	var resourceCollector *collector.Set
```

Comment:
```suggestion
	var operatorCollector *collector.Set
```

I think its better to use operatorCollector like we do in aws-operator. As the collector will have multiple metrics not just for resources.